festflow/templates/festflow/about.html

```diff
@@ -1,13 +1,34 @@
+{% load static %}
+
 <!DOCTYPE html>
 <html>
 <head>
-    <title>ABOUT PAGE</title>
+	<title>ABOUT PAGE</title>
```

Comment:
Just two spaces please, because, all other markup files use the similar indents.